api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -75,6 +78,27 @@ public void validate(Object target, Errors errors) {
 			errors.rejectValue("stopDatetime", "Visit.error.endDateBeforeStartDate");
 		}
 		
+		try 
+		{
+			Date birthDateTime = visit.getPatient().getBirthdate();
+			Date startDateTime = visit.getStartDatetime();
+			if(startDateTime.compareTo(birthDateTime) < 0) {
+			errors.rejectValue("startDatetime","Visit.error.startDateBeforeBirthDate"); 
```

Comment:
The formatting here is still incorrect. You need to run mvn clean install before committing as advised at https://wiki.openmrs.org/display/docs/Pull+Request+Tips